server/migrations/20221121091440_initialize.sql

```diff
@@ -0,0 +1,23 @@
+-- Add migration script here
+CREATE TABLE IF NOT EXISTS packager(
+  tg_uid INT NOT NULL UNIQUE PRIMARY KEY,
+  alias TEXT NOT NULL
+);
+
+CREATE TABLE IF NOT EXISTS pkg (
+  id INT AUTO INCREMENT PRIMARY KEY,
+  name TEXT NOT NULL,
+  assignee INT,
```

Comment:
We also need `assigned_at`

Reply:
Implement in 20e478155e80a8a8df2896ee0b74cedf55744492